kedro_serving/running_server.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 import pandas as pd
 from fastapi import FastAPI
+from kedro.framework.project import find_pipelines
```

Comment:
@cshaley I think the current modification is not the right one. The ``find_pipelines()`` function looks for ``create_pipeline`()`` functions in the project template, but any pipeline filtered / modified /created in the ``pipeline.py`` will not be loaded. I think the right way to load kedro pipelines is now : 

```python
from kedro.framework.project import pipelines
```